modules/nf-core/ngscheckmate/ncm/tests/main.nf.test

```diff
@@ -0,0 +1,184 @@
+nextflow_process {
+
+    name "Test Process NGSCHECKMATE_NCM"
+    script "../main.nf"
+    process "NGSCHECKMATE_NCM"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ngscheckmate"
+    tag "ngscheckmate/ncm"
+    tag "bedtools/makewindows"
+    tag "bcftools/mpileup"
+
+    setup {
+
+        run("BEDTOOLS_MAKEWINDOWS") {
+            script "../../../bedtools/makewindows/main.nf"
```

Comment:
@SPPearce @nvnieuwk : Will that work in a pipeline where bedtools isn't installed?  I'm having problems running this nf-test in Sarek where we don't have bedtools installed. The nf-test ran better after I installed bedtools in Sarek, but that doesn't seem like the right solution to me 🤔

Reply:
Hmmm.
The alternative would be to add files to test_data instead, but there are quite a few modules that do this kind of setup block. This is essentially taking a bed file and splitting it into individual bases.
I guess BCFTOOLS_MPILEUP exists in sarek already, so you didn't have an issue with that part.

Reply:
BCFTOOLS_MPILEUP is installed in Sarek, so that didn't cause any issues. However, I guess the nf-test should be kind of self-contained, and not rely on what other modules might or might not be installed in the particular pipeline that is trying to use the module.

I also had to change all `params.test_data..` to `params.modules_testdata_base_path..` to get the test kind of work, that is, I managed to get 3 out of 4 "subtests" working:

https://github.com/maxulysse/nf-core_sarek/pull/7